Cubical/Categories/Type.agda

```diff
@@ -8,9 +8,9 @@ open import Cubical.Categories.Category
 module _ ℓ where
   TYPE : Precategory (ℓ-suc ℓ) ℓ
```

Comment:
`TypePrecat`?

Reply:
Same as above, isn't all caps the standard naming?